cmake/modules/BuildFIO.cmake

```diff
@@ -2,14 +2,17 @@ function(build_fio)
   # we use an external project and copy the sources to bin directory to ensure
   # that object files are built outside of the source tree.
   include(ExternalProject)
+  if(ALLOCATOR)
+    set(FIO_EXTLIBS EXTLIBS=-l${ALLOCATOR})
+  endif()
   ExternalProject_Add(fio_ext
     DOWNLOAD_DIR ${CMAKE_BINARY_DIR}/src/
     UPDATE_COMMAND "" # this disables rebuild on each run
     GIT_REPOSITORY "https://github.com/axboe/fio.git"
-    GIT_TAG "bf0b7e75c1ccca4026c8880ed8a76fc7ef85f2f3"
+    GIT_TAG "53ee8c17adb00e3db4f2c9441777ba777390cb9f"
```

Comment:
does this pr depend on anything specific upstream, or is it just a general pullup?

Reply:
@cbodley I needed specific features to be on. Thank you for pointing out that I selected random commit, while releases are available. Changed to "fio-3.10".